x-pack/plugins/fleet/server/services/fleet_server/elastic_index.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from 'kibana/server';
+import hash from 'object-hash';
+
+import { FLEET_SERVER_INDICES } from '../../../common';
+import { appContextService } from '../app_context';
+import ESFleetAgentIndex from './elasticsearch/fleet_agents.json';
+import ESFleetPoliciesIndex from './elasticsearch/fleet_policies.json';
+import ESFleetPoliciesLeaderIndex from './elasticsearch/fleet_policies_leader.json';
+import ESFleetServersIndex from './elasticsearch/fleet_servers.json';
+import ESFleetEnrollmentApiKeysIndex from './elasticsearch/fleet_enrollment_api_keys.json';
+import EsFleetActionsIndex from './elasticsearch/fleet_actions.json';
+
+const INDEX_VERSION = 1;
+
+const FLEET_INDEXES: Array<[typeof FLEET_SERVER_INDICES[number], any]> = [
+  ['.fleet-actions', EsFleetActionsIndex],
+  ['.fleet-agents', ESFleetAgentIndex],
+  ['.fleet-enrollment-api-keys', ESFleetEnrollmentApiKeysIndex],
+  ['.fleet-policies', ESFleetPoliciesIndex],
+  ['.fleet-policies-leader', ESFleetPoliciesLeaderIndex],
```

Comment:
"policies" here is a bit confusing as we already have the concept of agent and integration policies within fleet. can you help me understand the purpose of these indices?

Reply:
I will try to explain what are these indices, first Fleet Server do not know about kibana saved object so all the data used by fleet server come from these indices:

`.fleet-policies` contains a full policy as it's going to be distributed to the agent (except the API keys)

`.fleet-policies-leader` it's not used in Kibana is used to have a policy coordinator per policy in Fleet server for future features.